docker-compose.yml

```diff
@@ -20,6 +20,7 @@ services:
       - zookeeper
     environment:
       KAFKA_BROKER_ID: 1
+      KAFKA_NUM_PARTITIONS: 3
```

Comment:
This is required now, because default partition count is 1 and this is not sufficient for most test cases, where subscriptions to multiple partitions are required.
And that is why most of tests are updated to publish messages to particular partition instead of topic - because kafka will load-balance messages between partitions and tests, consuming only one (first) partition would be failing.

Reply:
Makes sense